src/battery_manager_laptop.cpp

```diff
@@ -0,0 +1,66 @@
+#include <ros/ros.h>
+#include <sensor_msgs/BatteryState.h>
+#include <std_msgs/String.h>
+
+int percentage;
+
+void batteryCallback(const sensor_msgs::BatteryState::ConstPtr& msg)
+{
+    double capacity;
+    double charge;
+
+    if (msg->location == "hero2")
+    {
+        capacity = msg->capacity;
+        charge = msg->charge;
+        percentage = (charge/capacity)*100;
```

Comment:
The message already has a percentage field:
https://docs.ros.org/kinetic/api/sensor_msgs/html/msg/BatteryState.html and https://github.com/ros/common_msgs/blob/jade-devel/sensor_msgs/msg/BatteryState.msg#L40